### DISCUSSION TITLE
maxCells in getMarkerFeatures

### DISCUSSION THREAD

**Discussion by delysia** [GreenleafLab/ArchR#2022] [Questions / Documentation]
I'm doing pairwise testing using getMarkerFeatures to identify differential accessible peaks between case (8K cells) and control (3K cells) groups. With the same significance cutoff, I noticed that ArchR calls only 3 differential peaks when using the default setting of maxCells = 500, while ≥10K differential peaks were called if changing maxCells to 5000. 

I was wondering:
(1) if maxCells was capped at 500 by default purely out of concern for computing resources or there are other statistical considerations?
(2) if computing time/power is not a concern, would it be the most appropriate to set maxCells to Inf to allow all cells to be included in the analysis?

Also, to interpret my observation, does such dramatic difference in the number of differential peaks suggest the test was underpowered when using only 500 cells, or it suggests that the cell groups I was comparing were highly heterogeneous, or both?

Thank you for all your help and input!

**Comment by rcorces** [comment]
I think as long as the sizes of your groups arent super imbalanced, its ok to increase maxCells

**Reply by delysia** [reply]
Thank you for your quick response! Could I ask, in addition to limiting sample imbalance, are there any other considerations that went into setting the default maxCells to 500?
